src/main.rs

```diff
@@ -22,7 +22,7 @@ use std::fs::metadata;
 
 static ARG_INPUT_DIRECTORY: &'static str = "INPUT";
 static ARG_OUTPUT_DIRECTORY: &'static str = "output-directory";
-static DEFAULT_HTML_DIR: &'static str = "output";
+static DEFAULT_HTML_DIR: &'static str = "html";
```

Comment:
This change will create a merge conflict with #35, maybe we can submit this pull request earlier...